src/pages/library/common.js

```diff
@@ -452,32 +460,35 @@ export const SearchAndFilterComponent = ({
         }, ['clear'])
       ]),
       div({ style: { display: 'flex', alignItems: 'center' } }, [
-        h(DelayedAutoCompleteInput, {
-          style: { borderRadius: 25, flex: '1 1 0' },
-          inputIcon: 'search',
-          openOnFocus: true,
-          value: searchFilter,
-          'aria-label': `Search ${searchType}`,
-          placeholder: 'Search Name or Description',
-          itemToString: v => v[titleField],
-          onChange: onSearchChange,
-          suggestionFilter: _.curry((needle, { lowerName, lowerDescription }) => _.includes(_.toLower(needle), `${lowerName} ${lowerDescription}`)),
-          renderSuggestion: suggestion => {
-            return div({ style: { lineHeight: '1.75rem', padding: '0.375rem 0', borderBottom: `1px dotted ${colors.dark(0.7)}` } },
-              _.flow(
-                _.split(filterRegex),
-                _.map(item => _.toLower(item) === _.toLower(searchFilter) ? strong([item]) : item),
-                maybeMatch => {
-                  return _.size(maybeMatch) < 2 ? [
-                    _.truncate({ length: 90 }, _.head(maybeMatch)),
-                    div({ style: { lineHeight: '1.5rem', marginLeft: '2rem' } }, [...getContext(suggestion[descField])])
-                  ] : maybeMatch
-                }
-              )(suggestion[titleField])
-            )
-          },
-          suggestions: filteredData.data
-        }),
+        h(IdContainer, [id => h(Fragment, [
+          h(div, { id, className: 'sr-only' }, `Search ${searchType}`),
```

Comment:
Allows the label to be properly associated. Note that in the problematic case, Downshift generates and assigns an id for `aria-labelledby` even though one has been set with `aria-label`.

![image](https://user-images.githubusercontent.com/484484/201671305-9cbf1814-d0bd-4071-acf1-24142eb279d9.png)

vs.

![image](https://user-images.githubusercontent.com/484484/201671583-fc2a0751-13d7-4772-bfe5-764b38eb2968.png)

Reply:
I recommend you hide whitespace differences to view this.